### PR TITLE
ISSUE #3320 - Safeti base - Shapes

### LINKED ISSUE #3320: Safeti base - Shapes
### Description
Safeti base's UI is broken  in the viewer   in v5 - we need to update it to make it usable for users. This is for the shapes tab.

### Goals
- [ ]  As a user I want to use the shapes functionality from safetibase in v5

### Tasks
- [ ] Fix alignment of save and clear icons
image

- [ ] Fix in colour picker not showing the complete colour in the textfield
image

### Related Resources

[MockUp](https://www.figma.com/file/J5I3hYQTrLCjEVP1iqcyZK/3D-Viewer?node-id=136550%3A176546) 

### Note
**This is broken also in issues, so this should fix both**

### PR DESCRIPTION
**Pull request by The-Daniel**
This fixes #3320

#### Description
- Styles the shapes tab of viewer to fit V5 style
- Fix position of action buttons when editing shape name
- Fix padding in the hex value input inside the colour picker modal

### CODE REVIEW

**Review by george-hadfield — DISMISSED** [review]

File: frontend/src/v5/ui/routes/viewer/theme.ts [hunk 1]
```diff
@@ -134,6 +134,18 @@ export const theme = createTheme(
 							'&.react-autosuggest__suggestions-container': {
 								boxShadow: 'none',
 							},
+							'&.color-picker__panel': { // Colour picker modal
```
Comment (Amantini1997) [inline frontend/src/v5/ui/routes/viewer/theme.ts:137]: If you are writing the color picker style in the theme, could you please get rid of this file then ? `frontend\src\v5\ui\v4Adapter\dialogs\colorPickerDialog.overrides.ts`

It was used to tackle the color-picker style in the screenshot dialog, but if you put the style in a theme, it is no longer necessary

**Review by Amantini1997 — CHANGES_REQUESTED** [review]

File: frontend/src/v5/ui/routes/viewer/theme.ts (hunk 1 above)
Comment (The-Daniel) [reply frontend/src/v5/ui/routes/viewer/theme.ts:137]: Ah, I hadn't seen that file. Only the colorPicker.overrides file

**Review by Amantini1997 — APPROVED** [review]

**Pull request merged**